Dalamud/Utility/UnixSocketUtil.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.IO;
+using System.Net.Sockets;
+using System.Threading.Tasks;
+
+using Serilog;
+
+namespace Dalamud.Utility;
+
+/// <summary>
+/// A set of utilities to help manage Unix sockets.
+/// </summary>
+internal static class UnixSocketUtil
```

Comment:
This file should probably be in the Networking/Rpc folder since it seems pretty specific to RPCs, too much stuff in this one already